### DISCUSSION TITLE
Semantic segmentation without creating bounding boxes for object.

### DISCUSSION THREAD

**Discussion by Priyankapatil8** [ultralytics/ultralytics#4603] [Q&A]
Is a YOLOv8 Pretrained Model Achievable Without the Need for Bounding Box Annotations?
I want to semantic segmentation for every object presents in Dataset.

**Comment by suhailshaji** [comment]
Segmentation dataset required polygons only , segmentation format is : class x1 y1 x2 y2 ... xn yn  
refer the documentation [https://docs.ultralytics.com/datasets/segment](url)

**Reply by glenn-jocher** [reply]
@suhailshaji absolutely, YOLOv8 can handle semantic segmentation tasks without the need for bounding box annotations. Instead, you'll use polygon annotations to define the regions of interest in your images. Each object in your dataset will be represented by a series of points (x1 y1 x2 y2 ... xn yn) that outline the shape of the object, along with the class label.

For detailed guidance on how to prepare your dataset for segmentation with YOLOv8, please refer to our documentation on segmentation datasets. There you'll find the necessary format and instructions to get started. Happy segmenting! 😊🎨
